docs/adr/adr_statviz_frontend.md

```diff
@@ -0,0 +1,143 @@
+# [Statviz] What options of sharing should we allow and what comes with it?
+
+Decision Deadline:
+
+Discussion Participants: @haguesto, @aerinsol, maik
+
+## Status
+
+proposed
+
+## Context or Problem Statement
+
+In the DSEE 100x project titled "StatViz", our goal is to construct data visualizations derived from Boxtribute's data. This will allow our users to showcase their contributions and impact to the public through various mediums, such as their websites or newsletters.
+
+We are giving here an overview over possible options for the user to share these visualizations - csv, svg, jpeg, iframe - and how much development and maintenance effort is needed in order to decide which options we should offer we should offer.
+
+Currently, we are planning to build these visualizations in a React FE with a D3.js library and query the underlying data from a GraphQL endpoint.
+
+## Decision Drivers
+
+1. **Keep it simple**: Let's try to not add complexity to our codebase, especially for low maintenance sake.
+
+2. **Minimal development effort**: Keeping it simple might sometimes involve more effort to develop something.
```

Comment:
3. Create an effective foundation / re-usability for the planned future availability / aid ordering project 
4. Stay within DSEE 100x budget and avoid drawing from too many other resources (e.g. earmarked for other projects in Boxtribute) as much as possible